compiler/rustc_trait_selection/src/traits/select/confirmation.rs

```diff
@@ -1288,6 +1288,9 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                     ));
                 }
 
+                // FIXME(inherent_associated_types): Handle this case properly.
+                ty::Alias(ty::Inherent, _) => {}
```

Comment:
Might just be able to merge this case into the one above.

Reply:
We should always be able to normalized IATs, so should this be reachable? Maybe `bug` it? Or yeah, normalize.